src/lib/components/options/pages/General.svelte

```diff
@@ -31,7 +31,7 @@
 <Section title={i18n.getMessage('settingsAutosaveHeading')}>
   <Switch
     title={i18n.getMessage('settingsAutosave')}
-    description="Greatly reduce memory usage by not loading tab until selected"
+    description={i18n.getMessage('settingsAutosaveDescription')}
```

Comment:
```suggestion
```

Oops, seems like I forgot to remove this description while adding it to settings. This has nothing to do with autosave.